checkstyle-sonar-plugin/pom.xml

```diff
@@ -330,6 +330,11 @@
                 <branchRate>100</branchRate>
                 <lineRate>78</lineRate>
               </regex>
+              <regex>
+                <pattern>org.sonar.plugins.checkstyle.CheckstyleExecutor</pattern>
+                <branchRate>90</branchRate>
+                <lineRate>94</lineRate>
+              </regex>
```

Comment:
Yes, I know this problem, I did not recheck your coverage problem.
We experience this problem a lot even in main project - checkstyle. We keep code in old style while this is a problem in cobertura. Benefit that we receive by 100% code coverage is enormous so it do make sense.

Please downgrade your code to try-catch format. You can find examples on how to cover code of try-catch in main repo (powermock, ..... ) .